frontend/console/src/features/timeline/TimelineIcon.tsx

```diff
@@ -18,9 +18,10 @@ export const TimelineIcon = ({ event }: { event: Event }) => {
         return <Rocket01Icon className={`${style} ${textColor}`} />
       case 'log':
         return <LogLevelBadgeSmall logLevel={event.entry.value.logLevel} />
-      case 'ingress': {
+      case 'ingress':
         return <PackageReceiveIcon className={`${style} ${textColor}`} />
-      }
+      case 'cronScheduled':
+        return <AlarmClockIcon className={`${style} ${textColor}`} />
```

Comment:
I think I like the alarm clock icon, but also maybe a clock icon would be cool?

https://hugeicons.com/icon/clock-01-stroke-rounded
or maybe
https://hugeicons.com/icon/clock-05-stroke-rounded

Reply:
we should also update `eventTextColor(event)` and set a color for `cron` events